app/src/main/java/com/github/se/wanderpals/ui/screens/overview/Overview.kt

```diff
@@ -50,7 +51,12 @@ const val EMPTY_CODE = ""
  * @param navigationActions The navigation actions used for navigating to different screens.
  */
 @Composable
-fun Overview(overviewViewModel: OverviewViewModel, navigationActions: NavigationActions) {
```

Comment:
There is missing params in the documentation of Overview. You can add them I think.